app/scss/components/_nav.scss

```diff
@@ -62,14 +75,71 @@ nav {
   width: 100%;
   height: 96px;
 
-  .nav-links a {
-    color: $white;
-    padding: $default;
-    text-decoration: none;
+  @media (max-width: 750px) {
+    .nav-links {
+      overflow-y: scroll;
+      height: 100vh;
+    }
+  }
+}
+
+.nav-links a {
+  color: $white;
+  padding: $default;
+  text-decoration: none;
+
+  &.active {
+    // border-top: 4px solid rgba(255,255,255,0.9);
+    color: lighten($light-grey, 10%);
+  }
+
+  &:hover {
+    // border-top: 4px solid rgba(255,255,255,0.7);
```

Comment:
I agree that these top border things don't look good 😛   I think you can kill these comments.